go-controller/pkg/ovn/egressqos.go

```diff
@@ -357,6 +372,10 @@ func (oc *DefaultNetworkController) processNextEgressQoSWorkItem(wg *sync.WaitGr
 		return true
 	}
 
+	if err = oc.setEgressQoSStatus(eq, err); err != nil {
+		utilruntime.HandleError(fmt.Errorf("failed to update EgressQoS object %s with status: %v", eqKey, err))
```

Comment:
I see you only want to report error on final (10th) retry, is it intended? (i think for the other objects we report every time, but that is not a requirement)